gns3fy/gns3fy.py

```diff
@@ -1897,3 +1898,25 @@ def restore_snapshot(self, name=None, snapshot_id=None):
 
         # Update the whole project
         self.get()
+
+    def arrange_nodes(self):
+        """
+        Re-arrgange the existing nodes
+        in a geometric fashion (circle)
+
+        **Usage**
+
+        Project.arrange_nodes()
+        """
+        self.get()
+        if self.status != "opened":
+            self.open()
+
+        _r = 120
```

Comment:
This is the radius right? I think is better to have as an argument to the method with a default value like `def arrange_nodes_geometrically(self, radius=120)`

Reply:
I think the method name is too large, maybe
arrange_nodes_circular(self,  radius=120)